patchwork-events-world/src/main/java/net/minecraftforge/event/world/ChunkEvent.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.world;
+
+import net.minecraftforge.common.MinecraftForge;
+
+import net.minecraft.world.IWorld;
+import net.minecraft.world.chunk.Chunk;
+
+import net.patchworkmc.impl.event.world.WorldEvents;
+
+/**
+ * ChunkEvent is fired when an event involving a chunk occurs.<br>
+ * If a method utilizes this {@link net.minecraftforge.eventbus.api.Event} as
+ * its parameter, the method will receive every child event of this class.<br>
+ * <br>
```

Comment:
i think for now I'm not going to require changing the javadocs for Forge classes besides MCP-B-Gone because it just adds noise